### PR TITLE
TreeMap: update the 'Hierarchical Data Structure' demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Charts/HierarchicalDataStructure/description.md [hunk 1]
```diff
@@ -1 +1,29 @@
-The TreeMap component visualizes hierarchical data as&nbsp;a&nbsp;set of&nbsp;nested rectangles whose sizes are proportional to&nbsp;the visualized values. In&nbsp;this demo, the TreeMap accepts a&nbsp;data source of&nbsp;a&nbsp;hierarchical structure.
\ No newline at end of file
+The TreeMap component visualizes data as a set of rectangles (tiles). The tile size represents the data point value relative to other data points.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/HierarchicalDataStructure/description.md]:
```suggestion
The TreeMap component visualizes data as a set of rectangles (tiles). The tile size corresponds to a data point value relative to other data points. 
```
http://tw.devexpress.devx/language-guidelines/misused-words/represent/

File: JSDemos/Demos/Charts/HierarchicalDataStructure/description.md [hunk 2]
```diff
@@ -1 +1,29 @@
-The TreeMap component visualizes hierarchical data as&nbsp;a&nbsp;set of&nbsp;nested rectangles whose sizes are proportional to&nbsp;the visualized values. In&nbsp;this demo, the TreeMap accepts a&nbsp;data source of&nbsp;a&nbsp;hierarchical structure.
\ No newline at end of file
+The TreeMap component visualizes data as a set of rectangles (tiles). The tile size represents the data point value relative to other data points. 
+
+The TreeMap works with collections of objects. If objects in your collection have a plain structure, the component visualizes them as tiles. If your data is hierarchical, the TreeMap displays it as a group of nested tiles.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/HierarchicalDataStructure/description.md]: We should be able to address the TreeMap as "the component".

```suggestion
The TreeMap component works with collections of objects. If objects in your collection have a plain structure, the component visualizes them as tiles. If your data is hierarchical, the TreeMap displays it as a group of nested tiles.
```

File: JSDemos/Demos/Charts/HierarchicalDataStructure/description.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+
+To bind data to the component, pass the collection of objects to the TreeMap's [dataSource](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#dataSource) property.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/HierarchicalDataStructure/description.md]:
```suggestion
To bind data to the component, assign the collection of objects to the TreeMap's [dataSource](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#dataSource) property. 
```

File: JSDemos/Demos/Charts/HierarchicalDataStructure/description.md [hunk 4]
```diff
@@ -1 +1,29 @@
...
+
+To bind data to the component, pass the collection of objects to the TreeMap's [dataSource](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#dataSource) property. 
+
+Once you assign the data source, specify the [valueField](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#valueField) and [labelField](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#labelField). If you specify these properties, the component can determine the object fields that indicate TreeMap labels and values in the collection. The default values for these properties are `value` and `name`, respectively.
+
+If your data is hierarchical, you also need to specify the [childrenField](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#childrenField). The default value for the [childrenField](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#childrenField) is `items`. You can use these data field names to structure your collection as shown in this demo.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/HierarchicalDataStructure/description.md]:
```suggestion
If your data is hierarchical, you also need to specify the [childrenField](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#childrenField) property. The default [childrenField](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#childrenField) property value is `items`. You can use these data field names to arrange your collection as shown in this demo. 
```

File: JSDemos/Demos/Charts/HierarchicalDataStructure/description.md [hunk 5]
```diff
@@ -1 +1,29 @@
...
+            name: 'Sydney',
+            country: 'Australia',
+        }, {
+            value: 4440000,
+            name: 'Melbourne',
+            country: 'Australia',
+        }],
+    }
+
+This object produces a tile with **Australia** label. The **Australia** tile has two nested tiles labeled **Sydney** and **Melbourne**.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/HierarchicalDataStructure/description.md]:
```suggestion
This object produces a tile with the **Australia** label. The **Australia** tile has two nested tiles labeled **Sydney** and **Melbourne**.
```

File: JSDemos/Demos/Charts/HierarchicalDataStructure/description.md [hunk 6]
```diff
@@ -1 +1,29 @@
...
+
+To make TreeMap more informative, you can specify a [title](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/title/) and implement a [tooltip](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/tooltip/).
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/HierarchicalDataStructure/description.md]: Consistency.

```suggestion
To make the TreeMap more informative, you can specify a [title](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/title/) and implement a [tooltip](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/tooltip/).
```

**Review by dmitry-eliseev-devexpress — DISMISSED** [review]

File: JSDemos/Demos/Charts/HierarchicalDataStructure/description.md [hunk 7]
```diff
@@ -1 +1,29 @@
...
\ No newline at end of file
+The TreeMap component visualizes data as a set of rectangles (tiles). The tile size corresponds to a data point value relative to other data points. 
+
+The TreeMap component works with collections of objects. If objects in your collection have a plain structure, the component visualizes them as tiles. If your data is hierarchical, the TreeMap displays it as a group of nested tiles.
+
+To bind data to the component, assign the collection of objects to the TreeMap's [dataSource](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#dataSource) property. 
+
+Once you assign the data source, specify the [valueField](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#valueField) and [labelField](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#labelField). If you specify these properties, the component can determine the object fields that indicate TreeMap labels and values in the collection. The default values for these properties are `value` and `name`, respectively.
```
Comment (albertov05) [inline JSDemos/Demos/Charts/HierarchicalDataStructure/description.md]:
```suggestion
Once you assign the data source, specify the [valueField](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#valueField) and [labelField](/Documentation/ApiReference/UI_Components/dxTreeMap/Configuration/#labelField) properties. If you specify these properties, the component can determine the object fields that indicate TreeMap labels and values in the collection. The default values for these properties are `value` and `name`, respectively.
```

**Pull request merged**